app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -521,11 +492,10 @@ class ExplorationActivityTest {
 
   // TODO (#1855): Resolve ktlint max line in app module test
   // TODO(#89): The ExplorationActivity takes time to finish. This test case is failing currently.
+  @RunOn(TestPlatform.ROBOLECTRIC)
```

Comment:
@BenHenning The test case here needs to check the visibility of the pause button, but the button is only visible when the audio plays and it's an idle state. 

How should we do this? 
or it seems like a robolectric test only?

Reply:
We need a way to reasonably fake audio on Espresso. Is it possible to simulate different audio states through the fake audio library? E.g. could we tell the fake audio library to provide a playing state when the play button is clicked for both Robolectric & Espresso?

Reply:
https://github.com/oppia/oppia-android/issues/2255